co2eq/purchase/consumerpriceindices.yml

```diff
@@ -152,7 +140,7 @@ countries:
     2017: 103.5
     2018: 106.3
     2019: 109.5
-    2020: 110.5
+    2020: 112.6
```

Comment:
How was this available?

Reply:
It's available in the source for a few selected countries :) 
https://stats.oecd.org/Index.aspx?DataSetCode=PRICES_CPI#